net/cloudflared/files/cloudflared.config

```diff
@@ -1,9 +1,6 @@
-
 config cloudflared 'config'
 	option enabled '0'
 	option token ''
-	option config '/etc/cloudflared/config.yml'
```

Comment:
instead of removing these options, leave the value to empty, or comment out them